Clinical-Genomics/cgweb
# Upgrade dependencies

This PR fixes some deprecated dependencies

How to test:
1. Install on stage on clinical-db
1. open OP
1. log in
1. click around

Expected result:
Login should work
Changing views should work
